rows.go

```
@@ -224,7 +233,7 @@ func (rows *Rows) rowXMLHandler(rowIterator *rowXMLIterator, xmlElement *xml.Sta
 	if rowIterator.inElement == "c" {
 		rowIterator.cellCol++
 		colCell := xlsxC{}
-		_ = rows.decoder.DecodeElement(&colCell, xmlElement)
+		colCell.cellXMLHandler(rows.decoder, xmlElement)
```

golangci-lint (1.64.8)

Error return value of `colCell.cellXMLHandler` is not checked

(errcheck)